app/src/main/java/com/duckduckgo/app/di/VariantModule.kt

```diff
@@ -35,4 +39,8 @@ class VariantModule {
 
     @Provides
     fun weightedRandomizer() = WeightedRandomizer()
+
+    @Provides
+    fun appIconModifier(context: Context, appShortcutCreator: AppShortcutCreator): IconModifier =
```

Comment:
minor: this seems misplaced being in the `VariantModule`

Reply:
I didn't want to create a Module just for this, probably `SystemComponentsModule` would be better